packages/fx-core/tests/core/middleware/testAssets/v3Migration/jsSsoTab/expectedResult/js.app.yml

```diff
@@ -95,5 +95,5 @@ publish:
   - uses: teamsApp/publishAppPackage # Publish the app to Teams Admin Center (https://admin.teams.microsoft.com/policies/manage-apps) for review and approval
     with:
       appPackagePath: ./build/appPackage/appPackage.${{TEAMSFX_ENV}}.zip
-  # Output: following environment variable will be persisted in current environment's .env file.
-  # TEAMS_APP_PUBLISHED_APP_ID: app id in Teams tenant app catalog.
\ No newline at end of file
+    writeToEnvironmentFile:
+      publishedAppId: PUBLISHED_APP_ID
```

Comment:
@nliu-ms please help review teamsApp/publishAppPackage. The comment has been removed.

Reply:
Update it. Thanks.